src/index.js

```diff
@@ -0,0 +1,277 @@
+const tangramReference = require('tangram-reference').load();
+const color = require('./color.js');
+
+//getReferenceIndexedByCSS returns a reference based on ref indexed by the CSS property name instead of the reference original property name
+//it does a shallow copy of each property, therefore, it's possible to change the returned reference by changing the original one, don't do it
```

Comment:
Wouldn't be better to avoid shallow copy and do a deep copy?

Reply:
It will be certainly safer (if someone modifies it, bad things could happen) but slower. I left it like this because modifying a reference at runtime makes no sense to me, and if we need to modify it, the modification will be done in that same file because the reference is loaded there.
I would leave it like this.

Reply:
👌